packages/rancher-webhook/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.2.6/rancher-webhook-0.2.6.tgz
-version: 1.0.5
+url: https://github.com/rancher/webhook/releases/download/v0.2.7-rc2/rancher-webhook-0.2.7-rc2.tgz
```

Comment:
Is there a corresponding rancher-webhook ticket that describes what is getting updated or fixed in this version?

Reply:
this was discussed offline.